source/gettingstarted.rst

```diff
@@ -6,7 +6,7 @@ Getting Started
 Installation
 ------------
 
-In order to run :math:`\omega radlib`, you need to have a Python interpreter installed on your local computer, as well as a number of Python packages (`Dependencies`_). We recommend to install `Anaconda <https://www.anaconda.com/what-is-anaconda/>`_ as it installs Python, a number of required packages, and other useful tools (e.g. spyder).
+In order to run :math:`\omega radlib`, you need to have a Python interpreter installed on your local computer, as well as a number of Python packages (`Dependencies`_). We recommend you install `Anaconda <https://www.anaconda.com/what-is-anaconda/>`_ as it includes Python, numerous required packages, and other useful tools (*e.g.* `Spyder <https://www.spyder-ide.org/>`_).

```

Comment:
I would stick to "We recommend to install..." but have no strong opinion.

Reply:
To my knowledge common latin abbrevations like i.e. or e.g. do not necessarily need to be italicized. If we decide to change this here, we need to check the whole docs for consistency. @heistermann What do you think?

Reply:
> I would stick to "We recommend to install..."

This isn't grammatically-correct English, which is why I changed it. Another clean and grammatically correct alternative would be ``"We recommend installing..."; if you prefer that for some reason I'm happy to change it.

> If we decide to change this here, we need to check the whole docs for consistency. 

Great point; I didn't really think that through (especially since my intent both here and in #6 was not to affect anything related to style or content). I'd generally used that convention in my own writing, but its a bit of a holdover from a bygone era, as they have been well-accepted into the language in contemporary informal discourse. I'll revert it now.

Reply:
@CAM-Gerlach "We recommend installing..." sounds good. As non-native speaker it's always hard to get it all correct, thanks for addressing this.

Reply:
Sure; doesn't make any difference to me but if its clearer to a non-native reader, then I'm all for it. 

I'm the only English native on the Spyder team and have a experience in technical writing and tutoring non-native-English-speakers, so I tend to pick up quickly on the common quirks of such writing and can sometimes even tell someone's native language by how they write. On the other hand, I can sometimes use patterns, constructions and idioms that might be confusing or unfamiliar to a non-US audience, so I'm glad to have a diverse array of people from all around the word looking at my writing to make sure it makes sense.